exploration/01_baseline_calc_10yr_doy.py

```diff
@@ -0,0 +1,57 @@
+"""
+This script calculates the DOY baseline for FloodScan raster data at pixel
+level. It is currently in `exploratory` repo as it just uses GDRIVE `.nc` file
+to make local development more effecient (vs loading historical data from blob)
+
+In the official pipeline we can link it to the historical tif record on the
+blob.
+
+Steps:
+
+1. Smooth historical data w/ 10 day rolling mean (centered)
+2. filter data to last 10 years of data (exlcusive of current year)
+3. Calculate the mean SFED value per day of year from smoothed raster stack.
+"""
+
```

Comment:
Just in case this is useful- if you add a cell with
```
%load_ext autoreload
%autoreload 2
```
then you can make edits to your imported local functions (in `src`) and they will work right away in the notebook, instead of having to restart the kernel every time you make a change in the `src` file. Works nicely with the whole `pip install -e .` method.

Reply:
i had this, but it was causing issues w/ pre-commit. Have you seen that? I suppose some people have pre-commit ignoring `.ipynb` files, but since this is a `.py` file not sure the right way to deal with it?